RosNet.Test/RosNet.Test.csproj

```diff
@@ -5,6 +5,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="coverlet.msbuild" Version="3.0.3">
```

Comment:
I don't think this is necessary?

Reply:
the coverlet github action could not find the coverage file without it